bokehjs/src/lib/models/tools/gestures/box_zoom_tool.ts

```diff
@@ -8,6 +8,10 @@ import type {PanEvent, KeyEvent, TapEvent} from "core/ui_events"
 import {Dimensions, BoxOrigin} from "core/enums"
 import type {MenuItem} from "core/util/menus"
 import * as icons from "styles/icons.css"
+import {Enum} from "core/kinds"
+
+const ZoomOutGesture = Enum("double_tap")
+type ZoomOutGesture = typeof ZoomOutGesture["__type__"]
```

Comment:
Do we expect to support different possible gestures in the future?

Reply:
Possibly, but for simplicity of this PR I didn't implement anything new.